POH-README.md

```diff
@@ -1,24 +1,23 @@
-# Portable Open Hospital - POH
+# ![](./poh-bundle-linux-x64/oh.ico) OH - Open Hospital Portable | Client 

```

Comment:
the problem here is that POH-README.md is initially in the root of the repository, but after is copied in the distribution packages, so the hard-coded path doesn't work here. We could move the .ico file in the repository root and copy in the packages during the build or just duplicate, it might be useful in both locations

Reply:
Also, we can just replace replace `![](./poh-bundle-linux-x64/oh.ico)` with `![](./oh.ico)` after copied

Reply:
Up!

Reply:
I would not duplicate the file and keep it in the root directory only.

Reply:
```suggestion
# ![](./oh.ico) OH - Open Hospital Portable | Client 

```
so we can remove `oh.ico` from other subfolders and instruct the Makefile to copy it along with `POH-README.md`

Reply:
Will propose shortly a new PR including this